umbrella-framework/build.gradle.kts

```diff
@@ -27,6 +27,7 @@ kotlin {
             export(project(Modules.core))
             export(project(Modules.operationsApi))
             export(project(Modules.operationsStdlib))
+            export(project(Modules.utils))
```

Comment:
IMHO we don't need to export `utils` for the iOS framework. Because it works correctly without exporting explicitly.